runtime/plugin/osc52.lua

```diff
@@ -0,0 +1,26 @@
+if
+  vim.g.osc52 == false
+  or (
+    vim.g.osc52 == nil and (not os.getenv('SSH_TTY') or os.getenv('TMUX') or vim.o.clipboard ~= '')
```

Comment:
Do we also need to check if the TUI is attached? Or is `g:clipboard` already a TUI only concept?

Reply:
Makes sense. If we are embedded we don't want to write to stdout